Source/WebCore/rendering/RenderFlexibleBox.cpp

```diff
@@ -1511,7 +1511,7 @@ LayoutUnit RenderFlexibleBox::marginBoxAscentForFlexItem(const RenderBox& flexIt
     }
 
     if (isHorizontalFlow ? flexItem.isScrollContainerY() : flexItem.isScrollContainerX())
-        return std::clamp(*ascent, 0_lu, crossAxisExtentForFlexItem(flexItem)) + flowAwareMarginBeforeForFlexItem(flexItem);
+        return std::max(0_lu, crossAxisExtentForFlexItem(flexItem)) + flowAwareMarginBeforeForFlexItem(flexItem);
```

Comment:
Oh no, that's not what I was suggesting. What I was suggesting is this:
```
std::clamp(*ascent, 0_lu, std::max(0_lu, crossAxisExtentForFlexItem(flexItem))) + flowAwareMarginBeforeForFlexItem(flexItem);
```

Reply:
oh I get it now. Let me re-test.